data/campaigns/The_South_Guard/scenarios/06a_Tidings_Good_and_Ill.cfg

```diff
@@ -365,7 +365,7 @@
             message= _ "Ho! Me see elves. Elves tasty? Me take elves to eat."
         [/message]
         [message]
-            race=elf
+            speaker=second_unit
```

Comment:
Yeah, maybe. (It's because some languages have different words for "I" depending on the gender of the speaker. Not sure if any do the same with "we", but I wouldn't be too surprised.)